packages/suite/src/hooks/wallet/useCoinmarketExchangeOffers.ts

```diff
@@ -309,6 +309,10 @@ export const useOffers = ({ selectedAccount }: UseCoinmarketExchangeFormProps) =
                 selectedQuote.sendAddress,
                 sendStringAmount,
                 selectedQuote.partnerPaymentExtraId,
+                undefined,
+                undefined,
+                undefined,
+                ['broadcast', 'bitcoinRBF'],
```

Comment:
Have you considered other coins than bitcoin here?

Reply:
Should be fine, transaction functions check network type for bitcoinRBF.